hurumap_ke/tables.py

```diff
@@ -32,7 +32,8 @@
 FieldTable(['crimereport'], table_per_level=False, year = '2015')
 FieldTable(['typesofcrime'], table_per_level=False, year = '2015')
 FieldTable(['healthratios'], table_per_level=False, year = '2014')
-FieldTable(['voterregistration'], table_per_level=False, year = '2014')
+FieldTable(['voterregistration_2015'], table_per_level=False, year = '2014')
```

Comment:
In the previous data, it has "March 2015 and as at October 2015" and "December 2015"